features/docs/writing_support_code/world_namespaces.feature

```diff
@@ -0,0 +1,186 @@
+Feature: World Namespaces
```

Comment:
Looking at the existing features, it seems the existing `World` functionality was never properly documented in the acceptance tests.

Could you broaden the scope of this one, renaming it to just `world.feature` and adding two more scenarios:

- the one where you extend the world using a module
- the one where you define a custom class for the world (using a block)